packages/models-library/src/models_library/api_schemas_webserver/_base.py

```diff
@@ -70,3 +78,12 @@ def data_json(
             exclude_none=exclude_none,
             **kwargs,
         )
+
+
+#
+# mapping tools
+#
+
+
+def copy_dict(data: dict, update_keys: dict[str, str]) -> dict[str, Any]:
+    return {update_keys.get(k, k): v for k, v in data.items()}
```

Comment:
Q: why is this in a pydantic models-lib? or maybe why not using pydantic and dataclasses integrated copying mechanisms?
just a question, nothing to do.

Reply:
and this does some remapping of keys no?

Reply:
I can move it to another more general module